sdk/webpubsub/azure-messaging-webpubsub/src/test/resources/session-records/WebPubSubServiceClientTests.testBroadcastBytes.json

```diff
@@ -0,0 +1,20 @@
+{
+  "networkCallRecords" : [ {
+    "Method" : "POST",
+    "Uri" : "https://REDACTED.webpubsubdev.azure.com/api/hubs/test/:send?excluded=%5B%5D&api-version=2021-05-01-preview",
```

Comment:
maybe using Prod domain `webpubsub.azure.com`?

Reply:
We will do another recording later when the service is available in the public Azure portal.